flow360/component/simulation/translator/solver_translator.py

```diff
@@ -447,6 +447,60 @@ def porous_media_translator(model: PorousMedium):
     return porous_medium
 
 
+def bet_disk_entity_info_serializer(volume):
+    """BET disk entity serializer"""
+    v = convert_tuples_to_lists(remove_units_in_dict(dump_dict(volume)))
+    return {
+        "axisOfRotation": v["axis"],
+        "centerOfRotation": v["center"],
+        "radius": v["outerRadius"],
+        "thickness": v["height"],
+    }
+
+
+def bet_disk_translator(model: BETDisk):
+    """BET disk translator"""
+    model_dict = convert_tuples_to_lists(remove_units_in_dict(dump_dict(model)))
+    disk_param = {
+        "rotationDirectionRule": model_dict["rotationDirectionRule"],
```

Comment:
Have we ensured that all of these are not Optional?

Reply:
They are not